### PR TITLE
refine PR template

### PR DESCRIPTION
**Pull request by akeller**
## What is the purpose of the change

The PR Template, as it exists today, is working against everyone.

It's unclear why we ask for this information, no one is aligned on what "marketing activities" mean, and I see the same questions being asked about what should/shouldn't be in a PR. The template should help us with this.

## Are there related marketing activities

No, but I hope to one day brag on LinkedIn that we've figured out the best way to support components that release twice yearly and ones that release constantly/continuously. 

## When should this change go live?

After a healthy debate about whether people like this change or will actually use/read it.

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 ## When should this change go live?
 
-_Does this change need to be released before a certain date or milestone? If so, when?_
+<!-- PRs merged go to stage.docs.camunda.io first and must be manually released to docs.camunda.io. -->
+<!-- Help the DevEx team prioritize our work (reviews, merges, etc.) by opening PRs sooner. -->
+
+- [ ] There is no urgency with this change.
+- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
+- [ ] This functionality is already available but undocumented.
+- [ ] This is a bug fix or security concern.
```
Comment (christinaausley) [inline .github/pull_request_template.md:13]: Maybe something along the lines of "This change is not yet live and should not be merged until X (apply hold label)"?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,18 +1,22 @@
-## What is the purpose of the change
+## Description
```
Comment (christinaausley) [inline .github/pull_request_template.md:1]: Should we encourage users to apply appropriate labels and assign themselves as well?

**Review by christinaausley — APPROVED** [review]
Non-blocking comments -- THANK YOU for this!

File: .github/pull_request_template.md (hunk 2 above)
Comment (akeller) [reply .github/pull_request_template.md:1]: Great idea! I'll add a comment.

File: .github/pull_request_template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (RomanJRW) [reply .github/pull_request_template.md:14]: Not necessarily, but I also don't think I have a full picture. Something that might be worth doing is either having an agreed understanding of what urgency means or having the author clarify on urgency in the event that the first checkbox is left unchecked. I can imagine any of the following could be considered subjectively urgent, depending on the author:

- It needs releasing ASAP as it is security related
- It closes an L2 support ticket so helps maintain SLAs
- The feature is being released tomorrow, so the docs should be too

But only in the first scenario would I personally consider this "urgent"

**Review by pepopowitz — APPROVED** [review]
These are great changes 💪 🙌

File: .github/pull_request_template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (akeller) [reply .github/pull_request_template.md:14]: Each team seems to have its own definition of urgency, which I'm trying to respect here. With these specific questions I'm trying to get more at the _why_ and the _what_. The vast majority of PRs come in with "ASAP," "urgent", "immediately," etc. when in reality they seem to fall into these buckets. 

All of this to say, I'm ok with subjectivity as we balance between teams that release continuously and twice a year. 😄

Comment (akeller) [reply .github/pull_request_template.md:15]: Added!

**Review by christinaausley — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (RomanJRW) [reply .github/pull_request_template.md:15]: ha, sounds good. Thanks for clarifying! 💪

**Pull request merged**